src/cfml/system/modules/coldbox-commands/templates/modules/ModuleConfig.cfc

```diff
@@ -60,6 +60,8 @@ Optional Methods
 	this.modelNamespace		= "@modelNamespace@";
 	// CF Mapping
 	this.cfmapping			= "@cfmapping@";
+	// Auto-map models
+	this.autoMapModels		= false;
```

Comment:
This defaults to true.  Why do you want to set it to false in the templates?  We could change this to explicitly set it to true, but honestly I'm not sure how I feel about our templates have 17,000 default settings.  I kind of like keeping them minimalist to be honest.